src/05-derivatives/05-functional-derivatives.md

```diff
@@ -0,0 +1,390 @@
+# Functional derivatives
+
+## Functional derivatives maps
+
+A derivative map is a measure derived from a functional series, mapped onto spatial
+locations as defined by voxels in a volume or vertices on a surface.
+
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<label>/
+        func/
+            <source_entities>[_space-<space>][_res-<label>][_den-<label>][_desc-<label>]_<suffix>.{nii[.gz],gii,dscalar.nii}
+            <source_entities>[_space-<space>][_res-<label>][_den-<label>][_desc-<label>]_<suffix>.json
+```
+
```

Comment:
I always tend to forget what all these labels stand for, is there a place where they are listed? Maybe it could be good to include a link somewhere, or a short reminder

Reply:
The derivatives entities are going to be summarized in #645. Does that work?

Reply:
Yes! Very useful! Thanks :)